spec/services/target/create_target_services_spec.rb

```diff
@@ -40,5 +40,61 @@
         expect { subject }.to raise_error(MaxAllowTargetsError)
       end
     end
+
+    context 'when two targets match' do
```

Comment:
I suggest a similar context structure to the one you have in `spec/queries/target_query_spec.rb`